tools/theme/src/Theme.ts

```diff
@@ -55,8 +55,8 @@ export type Color =
     | 'dark-express'
     | 'darkest-express';
 export type Scale = 'medium' | 'large' | 'medium-express' | 'large-express';
-export type ThemeVariant = 'spectrum' | 'express';
```

Comment:
To make this "non-breaking" we'd need to continue to export `ThemeVariant`, in case anyone still consumes it.